src/pages/homepage.tsx

```diff
@@ -1,482 +1,212 @@
-                    'homepage:showcase.item2.link.title',
-                    'See Disinfo experiments with Open Terms Archive data'
-                  )}
-                >
-                  {t('homepage:showcase.item2.link.label', 'See')}
-                </a>
+                {t('homepage:cta_public.bloc3.link.label')}
               </LinkIcon>
-            </ShowcaseItem>
-          </ShowcaseList>
-        </Column>
-        <Column width={50} alignX="center" alignY="center">
-          <ButtonBlock
-            title={t(
-              'homepage:showcase.buttonblock.title',
-              'Have you developed an Open Terms Archive based tool?'
-            )}
-            desc={t('homepage:showcase.buttonblock.desc', 'Let the community know about it here')}
-            fillParent={true}
-          >
-            <Link
-              href={t('homepage:showcase.buttonblock.href', 'mailto:contact@opentermsarchive.org')}
-            >
-              <a title={t('homepage:showcase.buttonblock.link.title', 'Send us a mail')}>
-                <Button>{t('homepage:showcase.buttonblock.label', 'Contact us')}</Button>
-              </a>
-            </Link>
-          </ButtonBlock>
-        </Column>
+            </ButtonBlock>
+          </ButtonBlockList>
+        </Container>
       </Container>
-
-      {/* Partners */}
-      <Container layout="fluid" gridCols="12" gridGutters="11" flex={true} paddingX={false}>
-        <ThumbGalery
-          title={t('homepage:partners.title', 'Our Partners')}
-          subtitle={t('homepage:partners.subtitle', 'They make Open Terms Archive existing')}
-          titleLevel="h3"
-        >
-          <Link href="https://disinfo.quaidorsay.fr">
-            <a target="_blank">
-              <ThumbGaleryItem src="/images/logo-ambnum.png" width="158" height="80" />
-            </a>
-          </Link>
-          <Link href="https://disinformationindex.org/">
-            <a target="_blank">
-              <ThumbGaleryItem src="/images/logo-gdi.png" width="150" height="32" />
-            </a>
-          </Link>
-          <ThumbGaleryItem src="/images/nextgen.png" width="191" height="48" />
-          <ThumbGaleryItem src="/images/france-relance.png" width="77" height="77" />
-          <Link href="https://www.peren.gouv.fr/">
-            <a target="_blank">
-              <ThumbGaleryItem src="/images/logo-peren.png" width="150" height="121" />
-            </a>
-          </Link>
-        </ThumbGalery>
+      {/* Reuses */}
+      <Container gridCols="12" gridGutters="11">
+        <CardList title={t('homepage:reuses.title')} subtitle={t('homepage:reuses.subtitle')}>
+          <Card
+            image="/images/reuses/tosdr.jpg"
+            title="Terms of Service; Didn’t Read"
+            subtitle="Permet aux utilisateurs de donner collaborativement une note aux conditions d’utilisation selon le respect de leurs droits."
+            author="ToS;DR"
+            link="https://tosdr.org"
```

Comment:
This is hardcoded, it should be loaded from translation 😉

Reply:
there was a time when I had glasses and it was not so bad, fixed 😅